packages/homepage/content/docs/2-api.md

```diff
@@ -130,7 +130,7 @@ const url = `https://codesandbox.io/api/v1/sandboxes/define?parameters=${paramet
 
 #### Example Sandbox
 
-<iframe src="https://codesandbox.io/embed/6yznjvl7nw?editorsize=50&fontsize=14&hidenavigation=1&runonclick=1" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;" allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr" sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"></iframe>
+https://codesandbox.io/s/6yznjvl7nw?editorsize=50&fontsize=14&hidenavigation=1&runonclick=1
```

Comment:
It seems that something went wrong here

![Screenshot 2021-05-12 at 14 35 51](https://user-images.githubusercontent.com/4838076/117984001-5b358100-b32f-11eb-87ae-515675e6cfd5.png)

Reply:
This is just the same as is done in https://github.com/codesandbox/codesandbox-client/blob/master/packages/homepage/content/articles/this-month-april-2021/index.md and many other places: put the sandbox link on a single line

Reply:
humm interesting, I also wasn't able to figure out what is happening here, it should work fine. 
Want to give it a shot and fix it?

Reply:
Removed the dependency update, as you could do it in your Node 14 branch instead